easybuild/easyconfigs/i/impi/impi-2019.7.217-iccifort-2020.1.217.eb

```diff
@@ -12,11 +12,24 @@ source_urls = ['https://registrationcenter-download.intel.com/akdlm/irc_nas/tec/
 sources = ['l_mpi_%(version)s.tgz']
 checksums = ['90383b0023f84ac003a55d8bb29dbcf0c639f43a25a2d8d8698a16e770ac9c07']
 
+dependencies = [
+  # Uncomment this if your system UCX is too old.
+  #    ('UCX', '1.8.0'),  # needed by libfabric provider MLX introduced in Intel MPI v2019.6
+]
+
 dontcreateinstalldir = True
 
 components = ['intel-mpi', 'intel-psxe', 'intel-imb']
 
 # set up all the mpi commands to default to intel compilers
 # set_mpi_wrappers_all = True
 
+modextravars = {
+  # to enable SLURM integration with srun (site-specific)
+  # 'I_MPI_PMI_LIBRARY': 'libpmi2.so',
+
+  # set this if mpirun gives you a floating point exception (SIGFPE)
+  # 'I_MPI_HYDRA_TOPOLIB': 'ipl',
```

Comment:
Any documentation we can refer to for these?

Reply:
maybe this? https://software.intel.com/en-us/articles/intel-mpi-library-release-notes-linux
otherwise there is just the forums, including:
https://software.intel.com/en-us/forums/intel-clusters-and-hpc-technology/topic/807359

Reply:
I added a link to my own bug report in the end :)